### PR TITLE
Documentation improvements (and minor fixes)

### PR DESCRIPTION
**Pull request by hendrikweisser**
Added OpenSwathWorkflow to the UTILS page, improved the notes about mzid in tools that use idXML, made assorted small fixes and improvements (documentation, input/output parameters).

### CODE REVIEW

File: doc/doxygen/public/UTILS.doxygen [hunk 1]
```diff
@@ -80,8 +80,9 @@
 
  <b>Quantitation</b>
   - @subpage UTILS_ERPairFinder - Evaluate pair ratios on enhanced resolution (zoom) scans.
+  - @subpage UTILS_FeatureFinderSuperHirn - Find Features using the SuperHirn Algorithm (it can handle centroided or profile data, see .ini file).
   - @subpage UTILS_MRMPairFinder - Evaluate labeled pair ratios on MRM features.
-  - @subpage UTILS_FeatureFinderSuperHirn - Find Features using the SuperHirn Algorithm (it can handle centroided or profile data, see .ini file)
```
Comment (cbielow) [inline doc/doxygen/public/UTILS.doxygen]: why remove this?
FFSH is still there if i'm not mistaken

**Comment by cbielow** [comment]
+1 otherwise :)

**Comment by hendrikweisser** [comment]
> why remove this?
> FFSH is still there if i'm not mistaken

I haven't removed it, just moved it upwards so the tools are in alphabetical order.

**Comment by cbielow** [comment]
ouch. this happens if things are done in a hurry :)

**Pull request merged**